### DISCUSSION TITLE
embed openRefine

### DISCUSSION THREAD

**Discussion by michavardy** [pyscript/pyscript#1171] [Proposals]
Hey pyscript, 

at work we use Open-Refine: https://openrefine.org

OpenRefine is an open-source desktop application for data cleanup and transformation to other formats, an activity commonly known as data wrangling.

its great although it can't use pandas
I would like to stitch them together using pyscript 
open refine is a web server run on java
I just put the IP of open refine into my browser and it runs
is it possible to embed this program somehow into a site with a pyscript repl,

open refine can take json arguments

open refine can also output excel, csv or json datasets

I think that I could make them talk to each other, but I just don't want separate floating windows, I want them all in the same place.

anyone have any ideas?

**Comment by verhulstm** [comment]
i am interested in this. but it sounds like a long conversation. ping me and lets chat

**Comment by marimeireles** [comment]
I wonder if this discussion goes in the direction of what Dominik is tackling here?
https://github.com/pyscript/pyscript/issues/595#issuecomment-1265382412
